composer.json

```diff
@@ -25,22 +25,22 @@
         "php": ">=5.6"
     },
     "require-dev": {
-        "phpcq/all-tasks": "~1.1"
+        "phpcq/all-tasks": "~1.1",
+        "contao/core-bundle": "~4.4"
```

Comment:
We must require `contao/core-bundle` by require for dev, so can´t test this:
https://github.com/contao-community-alliance/url-builder/blob/release/1.3.0/src/Contao/CsrfUrlBuilder.php#L65